Resources/Storage/test/test_RFIOPlugIn.py

```diff
@@ -1,7 +1,7 @@
 """ test for RFIO plugin
```

Comment:
Pleeeeease, do not rename to a lower t ! Let's make a convention to start non valid yet tests with No_Test_<...>